src/app/(proper_react)/redesign/(public)/LandingView.test.tsx

```diff
@@ -24,9 +24,24 @@ import Meta, {
   LandingUsScanLimit,
 } from "./LandingView.stories";
 
-jest.mock("next-auth/react");
+jest.mock("next-auth/react", () => {
+  return {
+    signIn: jest.fn(),
+    useSession: jest.fn(() => {
+      return {};
+    }),
+  };
+});
 jest.mock("../../../hooks/useTelemetry");
 
+beforeEach(() => {
+  // For reasons that are unclear to me, the mock implementation defind in the
+  // call to `jest.mock` above forgets the implementation. I've spent way too
+  // long debugging that already, so I'm settling for this :(
```

Comment:
:cry:

Reply:
I think the reason might be that we are resetting mocks on every test run: https://github.com/mozilla/blurts-server/blob/MNTOR-3698-sign-in-button/jest.config.cjs#L145?

Reply:
Yeah, but that should be resetting them to the implementation defined in `jest.mock` (and just reset the values for `.mock.calls` and override calls to `.mockImplementation` or `.mockReturnValue`), as I understand it.